test-requirements.txt

```diff
@@ -1,2 +1,4 @@
 odoo-test-helper
 vcrpy
+odoo-addon-delivery_carrier_package_info @ git+https://github.com/OCA/delivery-carrier.git@refs/pull/983/head#subdirectory=delivery_carrier_package_info
+odoo-addon-stock_picking_batch_extended @ git+https://github.com/OCA/stock-logistics-workflow.git@refs/pull/1855/head#subdirectory=stock_picking_batch_extended
```

Comment:
pr closed and superssed on https://github.com/OCA/stock-logistics-workflow/pull/2120